src/forms/portfolio-share-form.schema.js

```diff
@@ -0,0 +1,67 @@
+import { componentTypes } from '@data-driven-forms/react-form-renderer';
+
+/**
+ * Creates a data-driven-form schema for sharing/un-sharing portfolio
+ */
+const newShareSchema = (rbacGroups, permissionVerbs) => (
+  { fields: [
+    {
+      component: 'sub-form',
+      title: 'Invite Group',
+      name: 'new_share',
+      key: '1',
+      fields: [{
+        name: 'group_uuid',
+        component: componentTypes.SELECT,
+        options: rbacGroups
+      }, {
+        name: 'permissions',
+        isRequired: true,
```

Comment:
if you want field to be required you have to add validate specification:
```javascript
import { componentTypes, validatorTypes } from '@data-driven-forms/react-form-renderer';
...
validate: [{
 type: validatorTypes.REQUIRED
}]
```

the `isRequired` key is just a visual prop for PF4 components